frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/DependencyRow.tsx

```diff
@@ -112,115 +108,82 @@ export const DependencyRow = ({ feature }: IDependencyRowProps) => {
-                                }
+                            {feature.dependencies[0]?.feature}
+                        </StyledLink>
+                        {checkAccess(UPDATE_FEATURE_DEPENDENCY, environment) ? (
+                            <DependencyActions
+                                feature={feature.name}
+                                onEdit={() => setShowDependencyDialogue(true)}
+                                onDelete={deleteDependency}
                             />
-                        </StyledMetaDataItemValue>
-                    </StyledMetaDataItem>
-                }
-            />
-            <ConditionallyRender
-                condition={
-                    hasParentDependency && !feature.dependencies[0]?.enabled
-                }
-                show={
-                    <StyledMetaDataItem>
-                        <StyledMetaDataItemLabel>
-                            Dependency value:
-                        </StyledMetaDataItemLabel>
-                        <span>disabled</span>
-                    </StyledMetaDataItem>
-                }
-            />
-            <ConditionallyRender
-                condition={
-                    hasParentDependency &&
-                    Boolean(feature.dependencies[0]?.variants?.length)
-                }
-                show={
-                    <StyledMetaDataItem>
-                        <StyledMetaDataItemLabel>
-                            Dependency value:
-                        </StyledMetaDataItemLabel>
-                        <VariantsTooltip
-                            variants={feature.dependencies[0]?.variants || []}
-                        />
-                    </StyledMetaDataItem>
-                }
-            />
-            <ConditionallyRender
-                condition={hasChildren}
-                show={
-                    <StyledMetaDataItem>
-                        <StyledMetaDataItemLabel>
-                            Children:
-                        </StyledMetaDataItemLabel>
-                        <ChildrenTooltip
-                            childFeatures={feature.children}
-                            project={feature.project}
-                        />
-                    </StyledMetaDataItem>
-                }
-            />
-
-            <ConditionallyRender
-                condition={Boolean(feature.project)}
-                show={
-                    <AddDependencyDialogue
+                        ) : null}
+                    </StyledMetaDataItemValue>
+                </StyledMetaDataItem>
+            ) : null}
+            {hasParentDependency && !feature.dependencies[0]?.enabled ? (
+                <StyledMetaDataItem>
+                    <StyledMetaDataItemLabel>
+                        Dependency value:
+                    </StyledMetaDataItemLabel>
+                    <span>disabled</span>
+                </StyledMetaDataItem>
+            ) : null}
```

Comment:
Not very important, but can we use `&&` instead of ternaries in this file to get rid of all the `: null`? It's kinda distracting when there's this many of them,  I think.

Reply:
We don't have reliable way of catching issues related to `&&` operator. https://docs.getunleash.io/contributing/ADRs/front-end/jsx-conditionals#pitfalls-of--operator